networkx/algorithms/centrality/group.py

```diff
@@ -28,7 +30,7 @@ def group_betweenness_centrality(G, C, normalized=True, weight=None, endpoints=F
 
     .. math::
 
-       c_B(C) =\sum_{s,t \in V-C; s\neq t} \frac{\sigma(s, t|C)}{\sigma(s, t)}
+       c_B(C) =\sum_{s,t \in V-C; s<t} \frac{\sigma(s, t|C)}{\sigma(s, t)}
```

Comment:
for undirected graphs, since we count the path from u to v as one, then the correct sign is < (only the path from s to t and not from t to s).
for directed graph you are correct and the correct sign in !=.
So, should I use the same equation as in the docstring of `betweenness_centrality` (without any sign?) and then explain the differences between directed and undirected in words?

Reply:
Yes, I think that may be the best approach -- to take the relationship between s and t out of the math and use words to explain its meaning in undirected and directed cases.
Thanks!